docs/guides/backends.rst

```diff
@@ -0,0 +1,184 @@
+
+``QuantumSimulator``\ s run in your own environment and therefore can be used without needing authentication credentials.
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # QuantumBackend creation example
+  :end-before: # End QuantumBackend creation example
+
+General Backend Methods
+-----------------------
+
+After intializing the backend we can send the ``Circuit`` and number of repetitions to the backend and read the measurement. For more information on creating circuits to send to a backend, refer to the :ref:`Circuits Guide <circuits_guide>`. For a single circuit, that is executed as follows:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # QuantumBackend run and measure circuit
+  :end-before: # End QuantumBackend run and measure circuit
+
+If you want to run multiple circuits with different number of samples, the code needs to be modified as follows:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # QuantumBackend run and measure circuitset
+  :end-before: # End QuantumBackend run and measure circuitset
+
+``QuantumBackend`` can also provide the distribution of the outcome measurement. It can be extracted using the following approach:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # Quantumbackend measurement distribution
+  :end-before: # End Quantumbackend measurement distribution
+
+In all instances, ``QuantumBackend`` automatically adds the measurement operators to the circuit. Therefore, the user is not required to include it in the circuit.
+
+Simulator-specific methods
+--------------------------
+
+``Quantumsimulator`` can be used to extract the wave function and expectation value of a given circuit. Below is an example of extracting these values using a ``Quantumsimulator`` called ``CirqSimulator``:
+
+.. literalinclude:: /examples/backends_guide.py
+  :start-after: # Quantumsimulator examples
+  :end-before: # End Quantumsimulator examples
+
+Zapata's backends
+=================
+
+SymbolicSimulator
+-----------------
+
+Zapata's ``SymbolicSimulator`` is a type of :ref:`quantum simulator <quantum_simulator>` that's built specifically to allow for the evaluation of circuits with :ref:`symbolic gates <symbolic_gates>`. Because there is extra overhead to allow for this broader type of evaluation, ``SymbolicSimulator`` is not the most performant simulator, but it can be useful for deeply understanding small circuits.
+
+The ``SymbolicSimulator`` can be used the same way any other simulator can be. To see more usage please refer to the :ref:`section in this guide on various methods <backend_methods>` or to the :doc:`API documentation </api/quantum/api/backend/index>`
+
+TrackingBackend
```

Comment:
I don't think `TrackingBackend` is a broader class of backends.
I would just drop the whole `TrackingBackend` thing from this convo and just leave it to be about `MeasurementTrackingBackend`.

Reply:
fixed in 4551c621e5c9c2b2295716c545eb6211b836df64